modules/olm-restricted-networks-configuring-operatorhub.adoc

```diff
@@ -97,6 +97,16 @@ operating system to mirror only the images that match. Valid values are
 <5> Optional: Only generate the manifests required for mirroring and do not actually
 mirror the image content to a registry.
 +
+.Example output
+[source,terminal]
+----
+using database path mapping: /:/tmp/190214037
+wrote database to /tmp/190214037
+using database at: /tmp/190214037/bundles.db <1>
+[...]
```

Comment:
We use "...", no brackets, to indicate that something is truncated.

Reply:
Whoops, bad habit. And I was even the one to update the guidelines w/ that. :sweat_smile: